packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -127,11 +133,22 @@ export function createReactExtension<
       const Result: any = (props: any) => {
         const app = useApp();
         const { Progress } = app.getComponents();
+        const mountPoint = data?.['core.mountPoint'] as
```

Comment:
This is a bit unfortunate and won't work if the mount point isn't defined through this `data` but a separate `attachComponentData` call. Let's not try to fix it now but could you add a TODO?